test/integration-test-wrapper.py

```diff
@@ -142,7 +143,6 @@ def main():
             ),
         ]),
         '--credential', f"journal.storage={'persistent' if sys.stderr.isatty() else args.storage}",
-        *args.mkosi_args,
```

Comment:
Why this is moved? Maybe, failure on rebase?

Reply:
This is intended, it needs to be before `--append` so that configuration files can override the arguments passed, specifically the list of configure scripts.